tests/model/test_model.py

```diff
@@ -723,6 +723,38 @@ def test_fail_get_messages(self, mocker, error_response,
         with pytest.raises(ServerConnectionFailure):
             model = Model(self.controller)
 
+    @pytest.mark.parametrize('response, expected_raw_content', [({
```

Comment:
Two things about the commit text:
* We could remove the `refactor:` label as this is not a refactor commit rather it adds a method.
* It's better to explicate what the commit would add/do in the commit title. In this case, we could mention that it would add support for fetching a raw message.